### DISCUSSION TITLE
Is 10 micron Firmware upgrade version 3.0.4 supported  by MW4 ?

### DISCUSSION THREAD

**Discussion by kaythriemer** [mworion/MountWizzard4#147] [Q&A] [answered]
Hi,
are there any experiences if Firmware upgrade version 3.0.4 is working with MW4  ?

I am a bit afraid to do the upgrade, because a downgrade back to 2.6 is not possible and I like MW4 and dont want to lose it.

Kind regards,
Kay

**Comment by mworion** [comment, marked as answer]
Hello Kay,
MW4 works with FW3.x there is even some more functions with v2 of MW4 and FW3. I currently run myself FW3.04 in my setup.
Michel

**Reply by kaythriemer** [reply]
Hello Michel, 
thank you for the information. I will upgrade and enjoy it :-)

Kay.
